modules/nw-cno-logs.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * networking/cluster-network-operator.adoc
+
+[id="nw-cno-logs_{context}"]
+= Viewing Cluster Network Operator logs
+
+You can view Cluster Network Operator logs by using the `oc logs` command.
+
+.Procedure
+
+* Run the following command to view the logs of the Cluster Network Operator:
```

Comment:
Even if it's just one step, it should be a numbered step

. Run

Reply:
@ahardin-rh, I think I've seen conflicting information on whether it's numbered or not.

Reply:
For mod-docs, we're using bullets.

Reply:
@jboxman Yeah, disregard. I missed that. Sorry! :woman_facepalming: